### ISSUE TITLE
uploading sketches or parts to github - adding feature to fritzing app

### ISSUE THREAD

**Issue by aquilanorica** [fritzing/fritzing-app#3064]
Hello fritzing team,

it would be a nice feature being able to upload sketches or parts to github for contribution right from the fritzing app such as "publish online".
Is there a recommended way or description to do so?

Thanks for your good work.

Keep on going.

Aquila

**Comment by failiz** [comment]
duplicate of #3752 and #835?

**Comment by KjellMorgenstern** [comment]
Yes, uploading sketches is #3752 and uploading parts is #835 . However, #835 only looks at the github toolchain. A huge number of users are not familiar with github, so they just upload parts to the forum, where they are eventually found or even kind of verified. 
I'll change this ticket to be about providing a way of easily creating a PR from a new part within Fritzing.

**Comment by KjellMorgenstern** [comment]
Add an option to the parts editor to create a pull request
